server/src/contact/contact.service.ts

```diff
@@ -1,13 +1,18 @@
 import {
   Injectable,
 } from '@nestjs/common';
-import { OdataService } from '../odata/odata.service';
-import { all, comparisonOperator } from '../odata/odata.module';
+import {
+  CrmService,
+} from '../crm/crm.service';
+import {
+  all,
+  comparisonOperator,
+} from '../crm/crm.utilities';
 
 @Injectable()
 export class ContactService {
   constructor(
-    private readonly odata: OdataService,
+    private readonly odata: CrmService,
```

Comment:
Prefer to change this reference to `crmService: CrmService` ?

Reply:
Yep! Ty